Lab2/ComputerStore.DataAccessLayer/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ConnectionStrings": {
+    "StoreConnection": "Server=(localdb)\\MSSQLLocalDB;Database=CoreComputerStore;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+  },
+  "SeedFolder": "..\\..\\..\\..\\ComputerStore.DataAccessLayer\\Seed"
```

Comment:
This path looks suspicious.
You can copy the Seed folder to the result of the build for now, and the path will look pretty.
Moreover, if you make a Seed folder to be a part of your project. It will be easier to modify from VS.